test/test.transporters.ts

```diff
@@ -113,29 +118,27 @@ describe('Transporters', () => {
     localUrlshortener = google.urlshortener('v1');
   });
 
-  it('should add headers to the request from params', () => {
-    testHeaders(localDrive);
-    testHeaders(remoteDrive);
+  it('should add headers to the request from params', async () => {
+    return Promise.all([testHeaders(localDrive), testHeaders(remoteDrive)]);
   });
 
-  it('should automatically add content-type for POST requests', () => {
-    testContentType(localDrive);
-    testContentType(remoteDrive);
+  it('should automatically add content-type for POST requests', async () => {
+    return Promise.all(
```

Comment:
question: is `Promise.all` necessary here? You could have awaited each of the test functions. Is it essential that these two tests run in parallel? This applies here and elsewhere the same pattern is used.

nit: don't use `async` unless the function needs to await.

Reply:
I updated them to use await